src/Utilities/AnsiBuilder.cs

```diff
@@ -0,0 +1,330 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace WheelMUD.Utilities
+{
+    /// <summary>
+    /// Mutable String class for Ansi based strings, optimized for speed and memory while retrieving the final result
+    /// as a string. Similar use to StringBuilder, but avoid a lot of allocations done by StringBuilder
+    /// </summary>
+    public class AnsiBuilder
+    {
+        /// <summary>
```

Comment:
nit: Prefer the condensed one-line form where it fits.
nit: Prefer to always end sentences in comments with punctuation.  (e.g. tooling which pulls from these doc comments read best with punctuation too.)